core/server/helpers/author.js

```diff
@@ -10,32 +10,30 @@
 // Block helper: `{{#author}}{{/author}}`
 // This is the default handlebars behaviour of dropping into the author object scope
 
-var proxy = require('./proxy'),
-    _ = require('lodash'),
-    SafeString = proxy.SafeString,
-    handlebars = proxy.hbs.handlebars,
-    templates = proxy.templates,
-    url = proxy.url;
+const {SafeString, templates, url, hbs: {handlebars}} = require('./proxy'),
```

Comment:
@kirrg001 is the standard for variable definitions in the core to use commas (i.e. `let a,b,c,d;`) over full lines
(i.e. `let a;
 let b;
 let c;
 let d;
`)?

If I remember correctly there were a couple of files in the core repo which use full lines; switching to es6 is a great time to standardize everything 😄

Reply:
Any ideas how the project wants to do this so I can make the change?

Reply:
Variable chaining (sometimes called multiple assignment) is something else, that's more like `one = two = three = 'test'`, and is generally a bad idea because of how JS handles scope in that case.

I don't know @kirrg001's preference for comma vs semi-colon assignment expressions but for reference in Ghost-Admin the rule is always semi-colon assignment (or restructuring) unless simply naming an unassigned variable. Eg:

```
let one = '1';
let two = '2';
let three, four;
```

Reply:
Oh, it looks like the example in the [codemods article](https://medium.com/airbnb-engineering/turbocharged-javascript-refactoring-with-codemods-b0cae8b326b9#052c) is wrong then